meetings/api/apps.py

```diff
@@ -1,7 +1,18 @@
 from __future__ import unicode_literals
-
 from django.apps import AppConfig
+import os
 
 
 class ApiConfig(AppConfig):
     name = 'api'
+
+
+class OsfOauth2AdapterConfig(AppConfig):
+    name = 'osf_oauth2_adapter'
+    # staging by default so people don't have to run OSF to use this.
+    osf_api_url = os.environ.get(
+        'OSF_API_URL', 'https://staging-api.osf.io').rstrip('/') + '/'
+    osf_accounts_url = os.environ.get(
+        'OSF_ACCOUNTS_URL', 'https://staging-accounts.osf.io').rstrip('/') + '/'
```

Comment:
These hard-coded values need to be in `meetings/settings/defaults.py`  probably empty and then define them in `local-dist.py` which is copied into `local.py`